web/pgadmin/misc/themes/__init__.py

```diff
@@ -5,9 +5,9 @@
 
 def get_all_themes():
     all_themes = {
-        "standard": {
-            "disp_name": "Standard",
-            "preview_img": "standard_preview.png"
+        "light": {
```

Comment:
you'll also have to change the default value for theme preference.

Reply:
Done